hs/src/Reach/Eval/Core.hs

```diff
@@ -2734,18 +2750,18 @@ evalPrim p sargs =
       let existingToks = st_toks st
       tokIsUniq <- tokIsUnique (map DLA_Var existingToks) $ DLA_Var tokdv
       doClaim CT_Assume tokIsUniq $ Just "New token is unique"
```

Comment:
This should be part of `trackToken`

Reply:
This assertion is specifically for assuming that the tokens we create via `new Token` are unique; it's not applicable to tracking tokens from an interact. Should I still move this to `trackToken` and conditionally run it for new tokens?